### PR TITLE
cleanup of consent rest api and allignment with sequence diagrams.

### PR DESCRIPTION
**Pull request by francis-pouatcha**
Just cleaned up the consent rest api package and had it aligned with the requirements. Provided readme so API can be viewed on swagger-ui. 

Could not write API file by hand as suggested by Denys. Will look at that later.

### CODE REVIEW

File: opba-consent-rest-api/src/main/java/de/adorsys/opba/consent/rest/api/domain/ConsentAuthTO.java [hunk 1]
```diff
@@ -6,97 +6,95 @@
 import io.swagger.annotations.ApiModelProperty;
 import lombok.Data;
 
+@SuppressWarnings("LineLength")
```
Comment (DG0lden) [inline opba-consent-rest-api/src/main/java/de/adorsys/opba/consent/rest/api/domain/ConsentAuthTO.java]: Would be nice if you put explanation in comment, why you suppress this warning and not fix it...

File: opba-consent-rest-api/src/main/java/de/adorsys/opba/consent/rest/api/resource/ConsentAuthorisationApi.java [hunk 2]
```diff
@@ -17,182 +17,148 @@
...
 
 /**
- * tags = "Authorization Entry Point", description = "Provides access to ConsentAPI"
+ * tags = "Authorization Entry Point", description = "Provides access to
+ * ConsentAPI"
  *
  * @author fpo
  */
-@Api(value = ConsentAuthorisationApi.PATH , tags = "Consent Authorization Api", description = "Entry point for processing a consent request redirected by the TppBankingApi over the FinTechApi to this ConsentAuthorisationApi.")
+@SuppressWarnings("LineLength")
```
Comment (DG0lden) [inline opba-consent-rest-api/src/main/java/de/adorsys/opba/consent/rest/api/resource/ConsentAuthorisationApi.java]: again, pls comment why it is suppressed

Comment (francis-pouatcha) [reply opba-consent-rest-api/src/main/java/de/adorsys/opba/consent/rest/api/resource/ConsentAuthorisationApi.java]: Ok. Will do.

Comment (francis-pouatcha) [reply opba-consent-rest-api/src/main/java/de/adorsys/opba/consent/rest/api/resource/ConsentAuthorisationApi.java]: Done. Build running.

**Pull request merged**